bse/bsesong.cc

```diff
@@ -681,7 +683,7 @@ SongImpl::remove_bus (BusIface &bus_iface)
   BseItem *child = bus.as<BseItem*>();
   BseUndoStack *ustack = bse_item_undo_open (self, __func__);
   // reset ::master-output property undoable
```

Comment:
The comment about undoable here can be removed.

Reply:
Fixed.